src/components/ReportWelcomeText.js

```diff
@@ -79,6 +82,7 @@ function ReportWelcomeText(props) {
                 )}
                 {isChatRoom && (
                     <>
+                        <Text>{props.translate('reportActionsView.beginningOfChatHistory')}</Text>
```

Comment:
This is sus, this is not in main

Reply:
Nice catch, removing! Maybe it was a merge conflict or something